src/test/regress/expected/citus_schema_distribute_undistribute.out

```diff
@@ -301,6 +301,21 @@ ERROR:  must be owner of table table5
 -- alter table owner, then redistribute
 SET role tenantuser;
 ALTER TABLE tenant1.table5 OWNER TO dummyregular;
+SET role dummyregular;
+SELECT citus_schema_distribute('tenant1');
+WARNING:  permission denied for database regression
+CONTEXT:  while executing command on localhost:xxxxx
+ERROR:  failure on connection marked as essential: localhost:xxxxx
+-- grant create on database, then redistribute
+SET role tenantuser;
+SELECT result FROM run_command_on_all_nodes($$ GRANT CREATE ON DATABASE regression TO dummyregular; $$);
```

Comment:
This is expected but looks like weird to me. Previously it would fail the same way during `CREATE SCHEMA`. Now for tenant tables, it fails during `citus_schema_distribute`. I am not sure about it.

Reply:
Preferably, we should only notice them when we use them in indirect way.